README.md

```diff
@@ -224,13 +249,24 @@ port 2376 when running secured using TLS. Please read section
 [Securing the Docker Compose ACAP using TLS](#securing-the-docker-compose-acap-using-tls) for
 more information.
 Below is an example of how to remotely run a docker command on an Axis device running
-the Docker Compose ACAP in unsecured mode:
+the Docker Compose ACAP in unsecured mode with TCP socket:
+
+With TCP Socket:
 
 ```sh
 DOCKER_INSECURE_PORT=2375
 docker -H=<device ip>:$DOCKER_INSECURE_PORT version
 ```
 
+With IPC Socket:
+
+Below is an example of how to remotely run a docker command on an Axis device running
+the Docker Compose ACAP in unsecured mode with IPC socket:
+
+```sh
+docker -H unix:///var/run/docker.sock  version
```

Comment:
This is accessing the IPC socket on your own machine - 
if you want to access the IPC socket on the device you need to ssh into the device and from there you can use`docker version` or `docker -H unix:///var/run/docker.sock version' 
I think you can skip this section - it does not add to the text